### PR TITLE
Be more clear on the hostname update

### PR DESCRIPTION
**Pull request by schaefi**
Try to make the hostname change more clear

### CODE REVIEW

**Comment by schaefi** [comment]
For reading compiled here:

https://w3.suse.de/~ms/Project/doc-susemanager/build/susemanager-cloud/single-html/susemanager-cloud/

File: xml/public-cloud-setup.xml [hunk 1]
```diff
@@ -110,14 +110,26 @@
       <para> Set the hostname to a name of your choice. Please note
        it is important to provide the <replaceable>system name</replaceable>
        and not the <replaceable>fully qualified host name</replaceable>
-       to the command. In the following example we use
-       <emphasis role="strong">suma</emphasis> as hostname and the
-       domain name is set to <emphasis role="strong">cloud.net</emphasis>.
-       </para>
-      <screen>$ hostnamectl set-hostname suma
-$ hostname -f
+       to the hostnamectl command.
```
Comment (aosthof) [inline xml/public-cloud-setup.xml]: I'd put hostnamectl in <filename> tags to be consistent.

File: xml/public-cloud-setup.xml [hunk 2]
```diff
@@ -110,14 +110,26 @@
...
+
+       On the assumption that the fully qualified host name of the
```
Comment (aosthof) [inline xml/public-cloud-setup.xml]: As I'm not sure if 'On the assumption that ...' is a common English phrase (Robert can tell for sure), I'd rather write 'Assuming that ..."

File: xml/public-cloud-setup.xml [hunk 3]
```diff
@@ -110,14 +110,26 @@
...
+       instance created in the cloud service is initially set to
+       <emphasis role="strong">cloud_instance.cloud.net</emphasis>,
+       this would result in <emphasis role="strong">cloud_instance</emphasis>
+       as the machine name and <emphasis role="strong">cloud.net</emphasis>
```
Comment (aosthof) [inline xml/public-cloud-setup.xml]: Here you write 'machine name', but above you called it 'system name'. Replace the one or the other to be consistent.

File: xml/public-cloud-setup.xml [hunk 4]
```diff
@@ -110,14 +110,26 @@
...
+       defining the domain name.
 
-suma.cloud.net</screen>
+       In the following example we will change the machine name to
```
Comment (aosthof) [inline xml/public-cloud-setup.xml]: Same here - machine name vs. system name

Comment (schaefi) [inline xml/public-cloud-setup.xml]: Thanks I've updated the inconsistencies

File: xml/public-cloud-setup.xml (hunk 2 above)
Comment (rjschwei) [inline xml/public-cloud-setup.xml]: We should say:

It is expected that the Fully Qualified Domain Name (FQDN) is set by the cloud framework, in this example we use cloud_instance.cloud.net

**Comment by rjschwei** [comment]
OK, just the wording change w.r.t. "assume" the rest looks good to me.

**Comment by schaefi** [comment]
Thanks I've update the docs. However I'm confused by Joel's last test report.

@keichwa: Please merge this once Robert or Alex gives us a thumbs up, thanks

**Pull request merged**